### ISSUE TITLE
Add Windows readme

### ISSUE THREAD

**Issue by mikedotexe** [near-examples/rust-high-level-cross-contract#10]
Since Docker requires Windows 10 Pro or Enterprise, we have not yet tested these commands out on it.
Run through this scenario with the proper system and update readme or create a new readme specific to Windows if need be.

**Comment by DiscRiskandBisque** [comment]
Agree, but wanted to note that Docker will run on Windows 10 Home via [the WSL2 backend](https://docs.docker.com/docker-for-windows/wsl/).
